README.md

```diff
@@ -261,3 +261,47 @@ If no length is specified, lengths for certain columns are defaulted:
 - *string*: `255`
 - *integer*: `11`
 - *biginteger*: `20`
+
+### Seeding your database
+
+As of 1.5.5, you can use the ``migrations`` shell to seed your database. This leverage the [Phinx library seed feature](http://docs.phinx.org/en/latest/seeding.html).
+By default, seed files will be looked for in the ``config/Seeds`̀  directory of your application.
+Please make sure you follow [Phinx instructions to build your seed files](http://docs.phinx.org/en/latest/seeding.html#creating-a-new-seed-class).
+
+As for migrations, a ``bake`` interface is provided for seed files:
+
+```bash
+# This will create a ArticlesSeed.php file in the directory config/Seeds of your application
+# By default, the table the seed will try to alter is the "tableized" version of the seed filename
+bin/cake bake seed Articles
+
+# You specify the name of the table the seed files will alter by using the ``--table`` option
+bin/cake bake seed Articles --table my_articles_table
```

Comment:
As per https://github.com/cakephp/cakephp/pull/7991/files#diff-56561cb12d6a5162f76fe28b07514e1cR202 it would be nice if we could also support `--model MyArticlesTable` IMO